x-pack/plugins/maps/public/connected_components/edit_layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -155,6 +162,58 @@ export function LayerSettings(props: Props) {
     );
   };
 
+  const renderShowLocaleSelector = () => {
+    if (!props.layer.supportsLabelLocales()) {
+      return null;
+    }
+
+    const options = [
+      {
+        text: i18n.translate('xpack.maps.layerPanel.settingsPanel.labelLanguageDefaultDropDown', {
+          defaultMessage: 'Default',
+        }),
+        value: DEFAULT_EMS_LOCALE,
```

Comment:
What is the default language? Is it just "english"? Do we need a "default" option. How about just selecting "english" when value is null?

Reply:
The Default language is what we currently use for the styles. The label is generally defined as `{name:latin}\n{name:nonlatin}` which is not always English. And in some countries in the Middle East and Eastern Asia native scripts are displayed in a new line under the Latin name.

Reply:
chatted offline and agreed to change "default" to "none" and move option to bottom of list.

Change change `DEFAULT_EMS_LOCALE ` to `NO_EMS_LOCALE `